packages/client/src/components/block-defaults/radio-block/config.tsx

```diff
@@ -0,0 +1,347 @@
+            }
+            renderInput={(params) => (
+                <TextField {...params} size="small" variant="outlined" />
+            )}
+            disableClearable
+        />
+    );
+};
+
+export const config: BlockConfig<RadioBlockDef> = {
+    widget: 'radio',
+    type: BLOCK_TYPE_INPUT,
+    data: {
+        style: {
+            padding: '4px',
+        },
+        value: 'yes',
+        label: 'Radio Group',
+        options: [
+            { label: 'Yes', value: 'yes' },
+            { label: 'No', value: 'no' },
+        ],
+        size: 'medium',
+        direction: 'row',
+        color: 'primary',
+        labelPlacement: 'end',
+        required: false,
+        disabled: false,
+    },
+    listeners: {
+        onChange: [],
+    },
+    slots: {
+        content: [],
+    },
+    render: RadioBlock,
+    icon: RadioButtonCheckedOutlinedIcon,
+    contentMenu: [
+        {
+            name: 'General',
+            children: [
+                {
+                    description: 'Options Management',
+                    render: ({ id }) => {
+                        const { data, setData } =
+                            useBlockSettings<RadioBlockDef>(id);
+                        const [numAdditionalOptions, setNumAdditionalOptions] =
+                            useState<string>('');
+                        const [optionLabels, setOptionLabels] =
+                            useState<string>('');
+
+                        const handleAddOptions = () => {
+                            const num = parseInt(numAdditionalOptions);
+                            if (num > 0 && optionLabels) {
+                                const labels = optionLabels
+                                    .split(',')
+                                    .map((label) => label.trim());
+                                const newOptions = labels
+                                    .slice(0, num)
+                                    .map((label) => ({
+                                        label,
+                                        value: label
+                                            .toLowerCase()
+                                            .replace(/\s+/g, '_'),
+                                    }));
+
+                                setData('options', [
+                                    ...(data.options as Array<{
+                                        label: string;
+                                        value: string;
+                                    }>),
+                                    ...newOptions,
+                                ]);
+                                setNumAdditionalOptions('');
+                                setOptionLabels('');
+                            }
+                        };
+
+                        // Find the current option object for the selected value
+                        return (
+                            <Stack spacing={2}>
```

Comment:
![image](https://github.com/user-attachments/assets/cb2aab23-e359-4948-bf62-c0e2fa63a51b)

We need to rethink this.  Avoid the extra step, of setting a number of options.

Why don't we just have a singular field. That allows you to add options dynamically without having to specify the amount of options. Something like this.

![image](https://github.com/user-attachments/assets/b5d70afa-dec2-4a75-8186-b67a4f559eb6)

The plus button would obviously add another option